readme.md

```diff
@@ -250,12 +250,12 @@ Useful documentation, examples, and recipes to get you started.
 
 The session from Google I/O 2018 covers the new performance engine, upcoming Lighthouse REST API, and using the Chrome UX report to evaluate real-user data.
 
-[![Lighthouse @ Google I/O 2018](https://img.youtube.com/vi/UvK9zAsSM8Q/0.jpg)](https://www.youtube.com/watch?v=UvK9zAsSM8Q)
+<p align= "center">[![Lighthouse @ Google I/O 2018](https://img.youtube.com/vi/UvK9zAsSM8Q/0.jpg)](https://www.youtube.com/watch?v=UvK9zAsSM8Q)</p>
```

Comment:
![image](https://user-images.githubusercontent.com/2301202/94814089-7b2abc80-03be-11eb-82ee-b60fcf1484f4.png)


you'll need to switch these to use `<img>` now that they're within other HTML